### ISSUE TITLE
New 'How To' help text required to explain search by grid square

### ISSUE THREAD

**Issue by CEJohnston** [nbnuk/nbnatlas-issues#650]
New 'How To' help text is required to explain search by grid square.  In particular to guide users in searching by 10km grid square.  This text to be linked to from the 'Explore by pre-defined areas' pages in all the Atlases.

**Comment by c-van-dierkson** [comment]
Have we considered this as part of #548 (which is now closed as part of  https://trello.com/c/Ne8H0YHh/217-convert-coordinates-to-grid-feasibility-research? 

@CEJohnston @sophiathirza

**Comment by sophiathirza** [comment]
@c-van-dierkson no, I think that this is something different. The new advanced records search has an option to search by grid square.

**Comment by CEJohnston** [comment]
@c-van-dierkson This is to do with providing users with more information about which records are returned by a grid square search.  I think at the time of writing the 10km grid was available to search against in the 'Explore by pre-defined areas' regions component but that is no longer the case.  There is now some useful text on the component (_Note: at present the region will only include a record supplied with a grid reference if the centre of the grid reference is within the boundary of the region._ ) which I think is enough to explain to users which records will get returned in a search against a region.  @sophiathirza I could not find the option to search by grid square in the advanced records search - do you think additional help text is required in this search option?
